packages/@repo/e2e/src/setup/datasets.setup.ts

```diff
@@ -0,0 +1,35 @@
+import {test as setup} from '@playwright/test'
+
+import {getClient} from '../helpers/clients'
+import {getE2EEnv} from '../helpers/getE2EEnv'
+import {sanityIdify} from '../helpers/sanityIdify'
+import {startTimer} from '../helpers/timer'
+
+const env = getE2EEnv()
+
+setup('setup test datasets', async () => {
+  const primaryDataset = sanityIdify(env.SDK_E2E_DATASET_0)
+  const secondaryDataset = sanityIdify(env.SDK_E2E_DATASET_1)
+
+  const client = getClient('primary')
```

Comment:
I'm probably misunderstanding something here, but would `'primary'` ever exist on setting up tests? And also where does the string `'primary'` come from?

Reply:
leftover from a older, worse way of identifying clients. didn't actually have any effect because that dataset name is never used, but fixed now!